admin_app/src/app/dashboard/api.ts

```diff
@@ -0,0 +1,23 @@
+const NEXT_PUBLIC_BACKEND_URL: string =
+  process.env.NEXT_PUBLIC_BACKEND_URL || "http://localhost:8000";
+
+type Period = "day" | "week" | "month" | "year";
```

Comment:
Can you export the type from `types.ts` so you don't have to rewrite the code?

Reply:
Good point! i'll fix this